src/tests/ftest/deployment/critical_integration.py

```diff
@@ -78,11 +82,13 @@ def test_passwdlessssh_versioncheck(self):
 
         for host in self.hostlist_clients:
             dmg_version_cmd = ("ssh -oNumberOfPasswordPrompts=0 {}"
-                               " 'dmg version -i'".format(host))
+                               " 'dmg version -i' -j".format(host))
```

Comment:
Minor - This works, but the command from the log is formatted strangely:
```
Running 'ssh -oNumberOfPasswordPrompts=0 wolf-129 'dmg version -i' -j'
```
```suggestion
                               " dmg version -i -j".format(host))
```